modules/fedimint-ln-client/src/db.rs

```diff
@@ -25,7 +23,7 @@ pub enum DbKeyPrefix {
     // Deprecated
     ActiveGateway = 0x28,
     PaymentResult = 0x29,
-    MetaOverrides = 0x30,
+    MetaOverridesDeprecated = 0x30,
```

Comment:
Why not just write a DB migration that deletes the key :sob: soon we'll have these unused DB prefixes all over the place.

Reply:
I am soon, we can also remove active gateway in same migration